tutor/resources/styles/components/task-plan/builder.less

```diff
@@ -75,14 +75,24 @@
   }
 }
 
+.-select-sections-btn {
+    border-radius: 4px;
+
+    &.invalid {
+        border: thin solid @form-error-color;
+        margin-right: 3px;
+    }
+}
+
 .due-before-open {
   line-height: 1.2rem;
   margin: -10px 0 30px;
 }
 
 .due-before-open,
 .readings-required,
-.problems-required {
+.problems-required,
+.required-hint {
   color: @form-error-color;
   font-size: 1.2rem;
 
```

Comment:
for `.problems-required` and `.readings-required`, there's a discrepancy between current and mockup:

## now:

![screen shot 2016-12-09 at 10 06 52 am](https://cloud.githubusercontent.com/assets/2483873/21055589/a5251032-bdf7-11e6-94b0-cc00c902a86a.png)

## mockup:

![screen shot 2016-12-09 at 10 07 00 am](https://cloud.githubusercontent.com/assets/2483873/21055588/a5202234-bdf7-11e6-971b-1ce876d6ab61.png)